frontend/admin/src/js/components/pool/ViewPool.tsx

```diff
@@ -112,6 +112,11 @@ export const ViewPoolPage = ({ pool }: ViewPoolPageProps): JSX.Element => {
     getSecurityClearance(pool.securityClearance ?? ""),
   );
 
+  const relativeToAbsoluteURL = (path: string): string => {
+    const { host } = window.location;
+    return `${host}${path}`;
```

Comment:
Not sure about this, but maybe we could add the protocol? I don't think it is required since we seem to upgrade all `http` to `https`.

```suggestion
    const { host, protocol } = window.location;
    return `${protocol}//${host}${path}`;
```

Reply:
I have no strong feelings either way, so, sure!